test/test_functional.py

```diff
@@ -31,6 +32,12 @@ class TestFunctional(unittest.TestCase):
     specgram = torch.tensor([1., 2., 3., 4.])
 
     test_dirpath, test_dir = common_utils.create_temp_assets_dir()
+
+    test_filepath_sinewave = os.path.join(test_dirpath, "assets", "sinewave.wav")
+    waveform_sinewave, sr_sinewave = torchaudio.load(test_filepath_sinewave)
+    E = torchaudio.sox_effects.SoxEffectsChain()
```

Comment:
Doing this means we should definitely clear the chain at the beginning of each test so we don't save much code. Let's create the chain inside in each test, so we are sure they do not interact.